### DISCUSSION TITLE
Error Page Layout

### DISCUSSION THREAD

**Discussion by lacorde** [nuxt/nuxt#16570] [Questions] [answered]
How to specify a layout for the [error.vue](https://v3.nuxtjs.org/getting-started/error-handling/) page ? (I would like to use my default layout)

The following snippet is ignored in error.vue :
```
definePageMeta({
  layout: "default",
});
```

I have a feeling this could be done with <[NuxtErrorBoundary](https://v3.nuxtjs.org/getting-started/error-handling/#rendering-errors-within-your-app)>, but can't find out how.

**Comment by thiswallz** [comment, marked as answer]
you can achieve this by using nuxt layout in your error page.

``` html
<template>
  <NuxtLayout>
    some page content
  </NuxtLayout>
</template>
```
https://v3.nuxtjs.org/api/components/nuxt-layout
